runtime/doc/eval.txt

```diff
@@ -6200,13 +6219,11 @@ rpcstart({prog}[, {argv}])				   {Nvim} *rpcstart()*
 			:let id = jobstart(['prog', 'arg1', 'arg2'], {'rpc': v:true})
 
 rpcstop({channel})					    {Nvim} *rpcstop()*
-		Closes an |RPC| {channel}.  If the channel is a job
-		started with |jobstart()|  the job is killed.
-		It is better to use |jobstop()| in this case, or use
-		|jobclose|(id, "rpc") to only close the channel without
-		killing the job.
-		Closes the socket connection if the channel was opened by
-		connecting to |v:servername|.
+		Deprecated. This function was used to stop a job with |rpc|
```

Comment:
It is backwards compatible so far that old code using rpcstart/rpcstop or calling rpcstop on remotely connected channels still should work as expected, but its semantics is very confusing together with the new channels design: it does something different if the channel happened to be a job or not. So the wording is deliberately somewhat stronger than eg `rpcstart`, which while inconsistent api, will not do anything unexpected when used together with the newer functions.